src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -127,6 +135,12 @@ using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 
+// Change host appropriately if you run your own Sentry instance.
+const host = "sentry.io";
```

Comment:
@bruno-garcia I have no idea if my C# changes actually work, did my best effort looking up docs. Could you please check it?

Thanks :)

C# rocks!

Reply:
Hmm we might as well align with https://github.com/getsentry/examples/blob/master/tunneling/dotnet/Program.cs

If the example here and in the `getsentry/examples` repo is supposed to be exactly the same, could we simply link to the example instead of copying the code here?

Reply:
+1 on linking to the `examples` repo.

Reply:
I thought this was just a quick copy paste for users but in the examples repo we can go more in-depth. In .NET we actually ship a package that includes this and all it takes to add os `UseSentryTunnel` to you request pipeline.

Using it on nuget trends:

https://github.com/dotnet/nuget-trends/blob/9a8fdc836b6e0f01dc44ced87495531aa89d066d/src/NuGetTrends.Web/Startup.cs#L100-L101

I'll fix the sample here.

Should we remove all samples here and only link to GitHub where we maintain this code?